README

```diff
@@ -1,6 +1,8 @@
 Network UPS Tools Overview
 ===========================
 
+image:https://travis-ci.org/networkupstools/nut.svg?branch=master["Build Status", link="https://travis-ci.org/networkupstools/nut"]
```

Comment:
At the moment, I don't think this will render on GitHub. See discussion as to why we haven't renamed it: https://github.com/networkupstools/nut/pull/226

Also, this ties the image to the master branch, which would be misleading for feature branches or released documentation (As I recall, README is included in the user manual.)